### ISSUE TITLE
Error when creating new directory

### ISSUE THREAD

**Issue by ecold** [samrat/jopbox#2]
When trying to create a new directory

``` clojure
(create-folder consumer access-token-response :dropbox "newdir")
```

I get the following error:

```
java.lang.ClassCastException: clojure.lang.PersistentArrayMap cannot be cast to [B
             core.clj:278 clj-http.core/request
             Var.java:415 clojure.lang.Var.invoke
           client.clj:663 clj-http.client/wrap-request-timing[fn]
          headers.clj:136 clj-http.headers/wrap-header-map[fn]
           client.clj:531 clj-http.client/wrap-query-params[fn]
           client.clj:548 clj-http.client/wrap-basic-auth[fn]
           client.clj:558 clj-http.client/wrap-oauth[fn]
           client.clj:571 clj-http.client/wrap-user-info[fn]
           client.clj:629 clj-http.client/wrap-url[fn]
           client.clj:197 clj-http.client/wrap-redirects[fn]
           client.clj:269 clj-http.client/wrap-decompression[fn]
           client.clj:412 clj-http.client/wrap-input-coercion[fn]
           client.clj:462 clj-http.client/wrap-additional-header-parsing[fn]
           client.clj:366 clj-http.client/wrap-output-coercion[fn]
           client.clj:149 clj-http.client/wrap-exceptions[fn]
           client.clj:490 clj-http.client/wrap-accept[fn]
           client.clj:504 clj-http.client/wrap-accept-encoding[fn]
           client.clj:480 clj-http.client/wrap-content-type[fn]
           client.clj:595 clj-http.client/wrap-form-params[fn]
           client.clj:619 clj-http.client/wrap-nested-params[fn]
           client.clj:578 clj-http.client/wrap-method[fn]
          cookies.clj:121 clj-http.cookies/wrap-cookies[fn]
             links.clj:50 clj-http.links/wrap-links[fn]
           client.clj:638 clj-http.client/wrap-unknown-host[fn]
           client.clj:753 clj-http.client/post
          RestFn.java:423 clojure.lang.RestFn.invoke
           client.clj:153 jopbox.client/create-folder
```

Has this been encountered before?

**Comment by mishadoff** [comment]
I think this issue can be closed. 
Do you plan to release next version with fixes?

**Comment by samrat** [comment]
Uploaded fixed version, 0.2.3 to Clojars. Thanks, @mishadoff !
